ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/processdetails.scala

```diff
@@ -62,6 +61,14 @@ object processdetails {
     def idWithName: ProcessIdWithName = ProcessIdWithName(ProcessId(id.toLong), ProcessName(name))
   }
 
+  sealed trait ProcessShapeFetchStrategy[ProcessShape]
+
+  object ProcessShapeFetchStrategy {
```

Comment:
We should think about unifying BasicProcess and BaseProcessDetails[Unit] - for now, let's add some comment/TODO for that?

Reply:
Added